taxclock/settings.py

```diff
@@ -40,3 +40,8 @@
 # Placeholder image when none found on scrape
 IMG_PLACEHOLDER = 'https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/\
                     placeholder.png'
+
+LOG_FILE = os.getenv('LOG_FILE', 'taxclock.log')
```

Comment:
Let's not use a log file for logging please. I'm unsure why you've changed the logging approach significantly here.